### ISSUE TITLE
Adding eslint-plugin-vuejs-accessibility to linter rules

### ISSUE THREAD

**Issue by kommunarr** [FreeTubeApp/FreeTube#1580]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
Currently, FreeTube's linter has little to no rules necessitating basic accessible design. Without accessibility rules in place, devs unknowingly utilize harmful anti-patterns and push out features that disabled users cannot benefit from. Irregularly addressing these built-up accessibility issues all at once is an extremely comprehensive, daunting, and time-consuming task. Instituting thorough linter accessibility rules will help keep devs accountable for their own code, lower sum development time, and improve the overall code quality of FreeTube.

**Describe the solution you'd like to see implemented**
[`eslint-plugin-vuejs-accessibility`](https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility) is an open source `eslint` plugin whose gamut of accessibility rules will massively improve the accessibility & code quality of FreeTube.

I strongly recommend that all of the recommended accessibility rules from `eslint-plugin-vuejs-accessibility` be added to FreeTube's `.eslintrc.js` as such:
```
{
  "rules": {
    "vuejs-accessibility/rule-name": "error"
  }
}
```
For newly required changes to the existing project that cannot be automatically applied, I would be glad to implement them all concomitantly in #1525. Alternatively, a separate PR can be made if needed.

**Describe alternatives you've considered**
1. [Less favorable] Implement most (but not all) of the accessibility rules. If there are certain pre-existing features that would require a difficult rework, implementing only most of the accessibility rules is a possible workaround (albeit not favorable).
2. Use a different open source project's accessibility linter rules.

**Additional context**
I would categorize this as a relatively high priority. Every new PR that does not follow accessibility guidelines is more work needed down the line. This is increasingly unsustainable, and at some point the duty of basic accessibility needs to shift onto devs' own code.
